db/tables/operations/alter.py

```diff
@@ -19,9 +18,9 @@ def rename_table(name, schema, engine, rename_to):
 
 
 def alter_table(table_name, table_oid, schema, engine, update_data):
-    if 'name' in update_data and 'sa_columns' in update_data:
+    if 'name' in update_data and 'columns' in update_data:
         raise ValueError('Only name or columns can be passed in, not both.')
     if 'name' in update_data:
         rename_table(table_name, schema, engine, update_data['name'])
-    if 'sa_columns' in update_data:
-        batch_update_columns(table_oid, engine, update_data['sa_columns'])
+    if 'columns' in update_data:
```

Comment:
Why change away from `sa_columns`?

Reply:
DRF serializers use the underlying source field as the key name. Since the serializer `columns` field refers to the `columns` field of the table model and not the `sa_column` field of the model, this change is needed.